transmission-remote-cli

```diff
@@ -1494,6 +1509,19 @@ class Interface(object):
                 server.remove_torrent_local_data(self.torrents[self.focus]['id'])
                 self.focus_next_after_delete()
 
+    def remove_finished_torrents(self, delete_local_data = False):
+        candidates = [torrent for torrent in self.torrents
+                if torrent['status'] == Transmission.STATUS_STOPPED
```

Comment:
I think the status should be compared to `Transmission.STATUS_SEED` or `Transmission.STATUS_SEED_WAIT`.  
I think `Transmission.STATUS_STOPPED` means the torrent was paused, but it may not be finished downloading yet.